benchmarks/data-serving/server/files/docker-entrypoint.sh

```diff
@@ -40,11 +40,9 @@ if [ "$1" = 'cassandra' ] || [ "$1" = 'bash' ]; then
 	sed -ri 's/(- seeds:) "127.0.0.1"/\1 "'"$CASSANDRA_SEEDS"'"/' "$CASSANDRA_CONFIG/cassandra.yaml"
 
 	for yaml in \
-		broadcast_address \
```

Comment:
Why are this line and the one that comes after it removed?

Reply:
Cassandra server was not starting with these parameters. Works after removing them. Need to debug
Do not merge PR

Reply:
So should these two stay removed?

Reply:
after reverting back to cassandra 3.11.6, we added these two again and tested. It's working fine.